scapy/route.py

```diff
@@ -10,9 +10,9 @@
 
 from __future__ import absolute_import
 
-
+import scapy.consts
 from scapy.config import conf
-from scapy.consts import WINDOWS, LOOPBACK_INTERFACE
+from scapy.consts import WINDOWS
```

Comment:
Could you remove this line and use `scapy.consts.WINDOWS` instead of `WINDOWS`?

Reply:
Well WINDOWS is guessed on loading only from inside const.py... Is it really necessary ?
Such code is used many times in scapy

Reply:
Importing `scapy.consts` **and** `WINDOWS` from `scapy.consts` makes loading this module load `scapy.consts` twice.

Reply:
What you could do maybe is using `from scapy import consts` and use `consts.WINDOWS` and `consts.LOOPBACK_INTERFACE` if you prefer?

Reply:
Fixed !